packages/framework/src/Component/Phing/PhingDownloader.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Component\Phing;
+
+use PharIo\Version\InvalidVersionException;
+use PharIo\Version\Version;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+class PhingDownloader
+{
+    public const string OPTION_VERSION = 'phing-version';
+    protected const string VERSION_URL = 'https://github.com/phingofficial/phing/releases/download/%s/phing-%s.phar';
```

Comment:
I know we were already discussing it but would not it be easier to use just `https://www.phing.info/get/phing-latest.phar`? You would not have to do the magic with the version resolving

Reply:
Nope as we want projects to be on locked versions to ensure compatibility.